libs/designsystem/src/lib/components/button/button.component.scss

```diff
@@ -70,8 +90,26 @@ $button-width: (
   --kirby-button-color: #{utils.get-color('black')};
   --kirby-button-border-color: #{utils.get-color('medium')};
 
+  // NOTE: This won't work!
+  //
+  // @include _button-hover(transparent, 'primary-contrast');
+  //
+  // Because transparent is not defined as a named color variant.
+  // Use the apply-hover() mixin directly instead.
+  @include interaction-state.apply-hover(
+    $background-color: transparent,
+    $color: utils.get-color($variant: 'black', $getValueOnly: true),
+    $property: '--kirby-button-background-color'
+  );
+
   &.destructive {
     --kirby-button-color: #{utils.get-color('danger')};
+
+    @include interaction-state.apply-hover(
+      $background-color: transparent,
+      $color: utils.get-color($variant: 'danger', $getValueOnly: true),
+      $property: '--kirby-button-background-color'
+    );
```

Comment:
Is it possible to define transparent as a named color variant such that we can avoid having to use `interaction-state.apply-hover` directly?

Reply:
@RasmusKjeldgaard and I already discussed this. The conclusion was, that it wouldn't make sense having a named color variant called "transparent".

Think about it this way: Which values would you expect `transparent-contrast`, `transparent-tint` and `transparent-shade` to have? 🤔😉